config/webpack/terra-dev-site.webpack.config.js

```diff
@@ -79,25 +103,49 @@ const devSiteConfig = (env = {}, argv = {}) => {
     console.log('Generated Aliases', alias);
   }
 
+  const indexEntryPoints = [];
+  indexEntryPoints.push('terra-dev-site');
+  // Strip the trailing / from the public path.
+  const basename = publicPath.slice(0, -1);
+
   return {
     entry: {
       'terra-dev-site': path.resolve(path.join(__dirname, '..', '..', 'lib', 'Index')),
+      rewriteHistory: path.resolve(path.join(__dirname, '..', '..', 'lib', 'rewriteHistory')),
+      redirect: path.resolve(path.join(__dirname, '..', '..', 'lib', 'redirect')),
     },
     plugins: [
-      new HtmlWebpackPlugin({
+      // terra-dev-site index.html
+      indexPlugin({
```

Comment:
maybe renaming this to `devSiteHtmlWepbackPlugin` could remove need for comment 🤷‍♀